attributes/default.rb

```diff
@@ -0,0 +1,6 @@
+
+ 
+default['export-node']['path'] = ENV['TEMP']
+default['export-node']['path'] = '/opt/kitchen/export-node' unless node['platform_family'] == 'windows'
```

Comment:
but typically `/tmp` is either cleared on boot or after files have not been accessed for a certain number of hours depending on distro. Either of those scenarios is not a typical test-kitchen scenario where this file would be created at the end of a test run and then accessed immediately at the beginning of verify.